htdocs/admin/inc/adminlist.inc.php

```diff
@@ -1,4 +1,18 @@
 <?PHP
+
+use EtoA\Admin\AdminRoleManager;
+use EtoA\Admin\AdminUserRepository;
+
+/** @var AdminUserRepository */
+$adminUserRepo = $app['etoa.admin.user.repository'];
+
+/** @var AdminRoleManager */
+$roleManager = $app['etoa.admin.role.manager'];
+
+index($adminUserRepo, $roleManager);
```

Comment:
method requires 3 arguments

Reply:
Hm why... its signature is

```
function index(AdminUserRepository $adminUserRepo, AdminRoleManager $roleManager): void
{
```

Reply:
hmm, apologies. Must be something off with the static analyzer which tries to match this to the index function in the alliances.php